.gitignore

```diff
@@ -3,3 +3,7 @@ build
 .vscode
 .idea
 *.iml
+
+camino-messenger-bot.yaml
```

Comment:
Because initially, before this PR there was only the file `camino-messenger-bot.yaml.example` and in the README it is saying to use this file and run the bot as `<binary_name> --config camino-messenger-bot.yaml`

So normally you would end up with this file (`camino-messenger-bot.yaml`) in the root of the repo.

Reply:
Ok, fine, I get it. You guys convinced me. BUT!
Lets add another two file-names to (without .example) gitignore as well. If we're doing it because of this reasoning, than lets not stop half-way.

Reply:
I've updated the README to use the two example config files for distributors and suppliers. And also updated the `.gitignore` file accordingly to only ignore these two config files. We don't need and don't mention the third one anymore.